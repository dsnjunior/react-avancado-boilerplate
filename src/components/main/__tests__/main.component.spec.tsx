import React from 'react'
import { render, screen } from '@testing-library/react'
import { Main } from '..'

describe('<Main />', () => {
  it('it renders the heading correctly', () => {
    const { container } = render(<Main />)

    expect(
      screen.getByRole('heading', { name: /react avanÃ§ado/i })
    ).toBeInTheDocument()
    expect(container.firstChild).toMatchSnapshot()
  })

  it('should render the colors correctly', () => {
    const { container } = render(<Main />)

    expect(container.firstChild).toHaveClass('bg-blue-400')
  })
})
