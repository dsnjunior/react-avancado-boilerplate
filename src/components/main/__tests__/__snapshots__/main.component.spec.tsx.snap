// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Main /> it renders the heading correctly 1`] = `
<main
  class="flex justify-center items-center h-screen bg-blue-400"
>
  <h1>
    React avan√ßado
  </h1>
</main>
`
